sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorConnection.java

```diff
@@ -48,9 +46,8 @@
     private final ConcurrentMap<String, SessionSubscription> sessionMap = new ConcurrentHashMap<>();
 
     private final AtomicBoolean isDisposed = new AtomicBoolean();
-    private final DirectProcessor<AmqpShutdownSignal> shutdownSignals = DirectProcessor.create();
-    private final FluxSink<AmqpShutdownSignal> shutdownSignalsSink = shutdownSignals.sink();
-    private final ReplayProcessor<AmqpEndpointState> endpointStates;
+    private final Sinks.One<AmqpShutdownSignal> shutdownSignalSink = Sinks.one();
```

Comment:
Looking at the usage of this is there a `Sinks.ony().replay()`? So we don't need to `cache()` when converting to a `Flux`.

Reply:
Unfortunately there is no replay. Or I would have. :(